CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ endif()
 # project name
 project(SFML VERSION 3.0.0)
 
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Isn't this already covered with `target_compile_features(SFML PUBLIC cxx_std_17)` further below?